packages/nc-gui/components/virtual-cell/barcode/Barcode.vue

```diff
@@ -26,6 +26,11 @@ const barcodeMeta = $computed(() => {
 
 const handleModalOkClick = () => (modalVisible.value = false)
 
+// const showBarcode = computed(() => barcodeValue?.value?.length > 0 && !tooManyCharsForBarcode.value)
+const showBarcode = computed(() => {
+  return cellValue?.value?.length > 0 && !tooManyCharsForBarcode.value
+})
```

Comment:
why not just 

```js
const showBarcode = computed(() => barcodeValue?.value?.length > 0 && !tooManyCharsForBarcode.value)
```

Reply:
It was a bit trickier as I thought. Please check if it's ok now.

Reply:
Now It's fine. Basically they are same. Just to keep it clean as `return` is implied in this way.